data/json/ui/widgets-structured/full_body_widget.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "//1": "Only used for displaying as a widget in the sidebar.  Not to be included with other bodygraphs.",
+    "//2": "Note: The code assumes the graph is at most 15x13 (WxH)",
+    "type": "body_graph",
+    "id": "full_body_widget",
```

Comment:
Currently, this replaces the original `full_body_widget` completely, for all sidebars. This is caused by hardcoded loading of `full_body_widget` in `widgets.cpp`. If this gets de-hardcoded, different body widget sized and looks would become possible.

However, with my limited understanding of the codebase, it seems to me that this requires another JSON type that "wraps" type `body_graph`.

Reply:
Solved this partially in 006ebb0, by adding field `body_part` to type `widget`.

A problem that remains is that the colorized graph is cached, which does not yet take into account that there can be different graphs.

Reply:
Solved caching in 7116dab, by adding the `graph_id` to the cache. Thus, it becomes invalid when another graph is used (through switching to another sidebar).